walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -70,25 +67,31 @@ public void optionsForNoAttendees() {
   public void noOptionsForTooLongOfARequest() {
     // The duration should be longer than a day. This means there should be no options.
     int duration = TimeRange.WHOLE_DAY.duration() + 1;
-    MeetingRequest request = new MeetingRequest(Arrays.asList(PERSON_A), duration);
+    MeetingRequest request = new MeetingRequest(Collections.singletonList(PERSON_A), duration);
```

Comment:
Same comment here.

Reply:
It would work i just thought it would be better since the arrays are going to be containing a single item